starcheck/calc_ccd_temps.py

```diff
@@ -169,16 +173,18 @@ def get_ccd_temps(oflsdir, outdir='out',
 
     make_check_plots(outdir, states, times,
                      ccd_temp, tstart, tstop, char=char)
-
     intervals = get_obs_intervals(sc_obsids)
-    obstemps = get_interval_temps(intervals, times, ccd_temp)
+    obsreqs = None
+    if orlist is not None:
+        obsreqs = dict([(o['obsid'], o) for o in read_or_list(orlist)])
```

Comment:
Use either ternary expression or full if-then-else to set `obsreqs`.   The `x = y; if c: x = z` pattern is less obvious when reading.